[2025-04-02T23:51:19.241+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-02T23:50:57.886361+00:00 [queued]>
[2025-04-02T23:51:19.248+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-02T23:50:57.886361+00:00 [queued]>
[2025-04-02T23:51:19.248+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-02T23:51:19.261+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Write_data_in_kafka> on 2025-04-02 23:50:57.886361+00:00
[2025-04-02T23:51:19.265+0000] {standard_task_runner.py:60} INFO - Started process 6187 to run task
[2025-04-02T23:51:19.268+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Write_data_in_kafka', 'manual__2025-04-02T23:50:57.886361+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmpa5d8owrt']
[2025-04-02T23:51:19.270+0000] {standard_task_runner.py:88} INFO - Job 62: Subtask Write_data_in_kafka
[2025-04-02T23:51:19.310+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-02T23:50:57.886361+00:00 [running]> on host 520607ad0646
[2025-04-02T23:51:19.381+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Write_data_in_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T23:50:57.886361+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-02T23:50:57.886361+00:00'
[2025-04-02T23:51:20.389+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-02T23:51:20.399+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Weather_Proj, task_id=Write_data_in_kafka, execution_date=20250402T235057, start_date=20250402T235119, end_date=20250402T235120
[2025-04-02T23:51:20.444+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-02T23:51:20.479+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
